### ISSUE TITLE
Allow to backup to WebDAV

### ISSUE THREAD

**Issue by beccon4** [ACINQ/eclair-mobile#126]
Currently Eclair tries to backup into Google Drive which is not available on LineageOS. So I get a confusing flashing error message "No Backup". 

I'd rather use my private cloud (like e.g. Nextcloud) to save my backups - so WebDAV support instead relying  of proprietary interfaces would be nice.

**Comment by karloluiten** [comment]
Agreed, webdav would be very welcome. I also think the privacy-friendly self-hosted systems fit better into the "ecosystem" of bitcoin/LN (as opposed to Google)

**Comment by pm47** [comment]
That's a great idea. Google Drive is definitely a tradeoff (a pretty good one imho), and we designed the thing to allow multiple backends. @dpad85 is on vacation currently but we'll take a look at that when he gets back

**Comment by pm47** [comment]
@dpad85 ping

**Comment by zatJUM** [comment]
+1
